setup.py

```diff
@@ -16,10 +16,8 @@
 
 
 def read(*names, **kwargs):
-    return io.open(
-        join(dirname(__file__), *names),
-        encoding=kwargs.get('encoding', 'utf8')
-    ).read()
+    with io.open(join(dirname(__file__), *names), encoding='utf-8') as fp:
+        return fp.read()
```

Comment:
Maybe it's time to also get rid of the `**kwargs` - I don't think I have ever needed a different encoding.